packages/flutter/lib/src/animation/animation_controller.dart

```diff
@@ -170,6 +197,17 @@ class AnimationController extends Animation<double>
   /// identifying animation controller instances in debug output.
   final String debugLabel;
 
+  /// The behavior of the animation or simulation when animations are disabled.
+  ///
+  /// Defaults to [AnimationBehavior.normal] for the [new AnimationBehavior()]
```

Comment:
I didn't know that [new Foo] was required to identify a constructor. It  looks (grepping through our sources) like the '()' isn't needed, so just [new AnimationBehavior]

Reply:
Fixed